Sources/BlueTriangle/NetworkCapture/CapturedRequestCollector.swift

```diff
@@ -56,16 +59,27 @@ final class CapturedRequestCollector: CapturedRequestCollecting {
         }
     }
 
-    func start(page: Page) {
-        self.spanStartTime = self.timeIntervalProvider()
-        self.timerManager.start()
+    func start(timer: BTTimer, timerRequestBuilder: @escaping (BTTimer) throws -> Request) {
```

Comment:
I don't love passing the timer request builder this way and am open to suggestions for alternatives

Reply:
I cannot think of a better alternative atm.